doc/man3/EVP_DigestSignInit.pod

```diff
@@ -118,7 +118,8 @@ EVP_DigestSignUpdate() hashes I<cnt> bytes of data at I<d> into the
 signature context I<ctx>. This function can be called several times on the
 same I<ctx> to include additional data.
 
-EVP_DigestSignFinal() signs the data in I<ctx> and places the signature in I<sig>.
+EVP_DigestSignFinal() can be used to query the maximum size of the signature
+or to actually sign the data in I<ctx> and places the signature in I<sig>.
```

Comment:
May I suggest instead: `Unless I<sig> is NULL EVP_DigestSignFinal() signs the data in I<ctx> and places the signature in I<sig>. Otherwise the maximum necessary size of the output buffer is written to ...`

Reply:
Done.